packages/jazz-tools/src/tests/schema.test.ts

```diff
@@ -163,6 +163,35 @@ describe("co.json TypeScript validation", () => {
       }>();
   });
 
+  /* Special case from reported issue:
+   ** See: https://github.com/garden-co/jazz/issues/1496
+   */
+  it("should apply the same validation to optional json [JAZZ-1496]", async () => {
+    interface ValidInterface0 {
+      value: string;
+    }
+    interface ValidInterface1 {
+      value: string | undefined;
+    }
+    interface InvalidInterface {
```

Comment:
nit: this interface is technically valid

What about renaming the interface to `InterfaceWithOptionalTypes`?